docs/cn_doc/cn-contribute-to-documentation.md

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
you can make a label --- not ready for review -- I'll go through but others can ignore.